assets/components/ctaCircle/ctaCircle.jsx

```diff
@@ -13,7 +13,7 @@ type PropTypes = {
   text: string,
   modifierClass: ?string,
   url?: ?string,
-  onClick?: () => void,
+  onClick?: ?() => void,
```

Comment:
This looks quite confusing. Is it possible to be more explicit?

Reply:
Done.